labgrid/driver/rawnetworkinterfacedriver.py

```diff
@@ -59,7 +59,7 @@ def start_record(self, filename, *, count=None, timeout=None):
         Starts tcpdump on bound network interface resource.
 
         Args:
-            filename (str): name of a file to record to
+            filename (str): name of a file to record to, or "-" to record to stdout
```

Comment:
Perhaps use filename=None for that case? "-" isn't really a convention in Python.

Reply:
Done